### PR TITLE
[ImportExport]Release Api 2021-01-01 version

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

Release Api 2021-01-01 version.

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/ImportExport/.gitignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+tools
+custom/*.psm1
+custom/autogen-model-cmdlets
+test/*-TestResults.xml
+/*.ps1
+/*.ps1xml
+/*.psm1
+/*.snk
+/*.csproj
+/*.nuspec
```
Comment (Nickcandy) [inline src/ImportExport/.gitignore:16]: Should we add git attributes and gitignore file here?

Comment (LucasYao93) [reply src/ImportExport/.gitignore:16]: This is automatically generated by autorest powershell.

File: src/ImportExport/Az.ImportExport.psd1 [hunk 2]
```diff
@@ -10,9 +10,8 @@
   PowerShellVersion = '5.1'
   DotNetFrameworkVersion = '4.7.2'
   RequiredAssemblies = './bin/Az.ImportExport.private.dll'
-  RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.2.8'; })
   FormatsToProcess = './Az.ImportExport.format.ps1xml'
-  FunctionsToExport = 'Get-AzImportExport', 'Get-AzImportExportBitLockerKey', 'Get-AzImportExportLocation', 'New-AzImportExport', 'New-AzImportExportDriveListObject', 'Remove-AzImportExport', 'Update-AzImportExport'
+  FunctionsToExport = 'Get-AzImportExport', 'Get-AzImportExportBitLockerKey', 'Get-AzImportExportLocation', 'New-AzImportExport', 'New-AzImportExportDriveListObject', 'Remove-AzImportExport', 'Update-AzImportExport', '*'
```
Comment (Nickcandy) [inline src/ImportExport/Az.ImportExport.psd1:14]: Is this added manually? Why do we need a wildcard here?

File: src/ImportExport/README.md [hunk 3]
```diff
@@ -45,10 +45,11 @@ In this directory, run AutoRest:
 > see https://aka.ms/autorest
 
 ``` yaml
+branch: a59006e985447b922b68210244e5fd024a30f1b4
 require:
   - $(this-folder)/../readme.azure.noprofile.md
-input-file:
-  - $(repo)/specification/storageimportexport/resource-manager/Microsoft.ImportExport/stable/2016-11-01/storageimportexport.json
+  - $(repo)/specification/storageimportexport/resource-manager/readme.md
```
Comment (BethanyZhou) [inline src/ImportExport/README.md:51]: why does ImportExport take a readme.md as input file rather than json files?

Comment (LucasYao93) [reply src/ImportExport/README.md:51]: The readme.md file defines the supported swagger json versions. it's maintained by the service team.
If the service team asks us to upgrade the Api version. No need to replace swagger json on azure powershell side.
This method should be preferred

File: src/ImportExport/Az.ImportExport.psd1 (hunk 2 above)
Comment (LucasYao93) [reply src/ImportExport/Az.ImportExport.psd1:14]: No, It's generated by autorest.powershell.

File: src/ImportExport/README.md (hunk 3 above)
Comment (BethanyZhou) [reply src/ImportExport/README.md:51]: Sounds great. But in fact, we still need to refresh the value for branch tag.

**Review by BethanyZhou — APPROVED** [review]
Looks good to me

**Comment by BethanyZhou** [comment]
@Nickcandy please help move this PR to main branch

**Pull request merged**